src/packages/@ncigdc/utils/ajax/index.js

```diff
@@ -35,11 +35,25 @@ export function fetchAuth(options: { endpoint: string }): Object {
 }
 
 // $FlowIgnore
-export const fetchApi = (endpoint, opts = {}) => {
+export const fetchApi = (endpoint, options = {}) => {
```

Comment:
I think `opts` is missing

Reply:
Hmm must've gotten lost somewhere along the way (merges etc.)

Reply:
The opts come from line 47 instead ;)

Reply:
oh I see what you did there

:+1: